Realme/RE513CL1/res/values/notch.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Height of the status bar in portrait -->
+    <dimen name="status_bar_height_landscape">110.0px</dimen>
+    
+    <dimen name="quick_qs_offset_height">@dimen/status_bar_height_portrait</dimen>
+
+    <!-- Height of the status bar in landscape -->
+    <dimen name="status_bar_height_portrait">110.0px</dimen>
```

Comment:
It sounds pretty weird to have the same in  landscape and portrait, got photos?

Reply:
This device has a punch hole display.
I just used whatever values the overlay apks had